wiki/LocalSettings.php

```diff
@@ -132,6 +132,13 @@
 $wgGroupPermissions['*']['edit'] = false;
 $wgGroupPermissions['*']['read'] = false;
 
+
+## Search more than just articles
+$wgNamespacesToBeSearchedDefault[NS_CATEGORY] = true;
+$wgContentNamespaces = [ NS_MAIN, NS_HELP, NS_PROJECT, NS_CATEGORY ];
```

Comment:
Går det även att ta med User här?

Reply:
ref: https://www.mediawiki.org/wiki/Manual:Namespace_constants

Reply:
Rimligt! Lägger in det när jag kommer hem

Reply:
Jag tror jag gjorde en PR på din PR-branch

Reply:
Fick det att funka lokalt.

Tror man behöver köra `php updateArticleCount.php --update` från /var/www/html/maintenance i containern för att det ska få effekt.
men kan ha varit någon annan cache som spökat.

om 2:an är med i `WHERE page_namespace IN ('0','12','4','14','2')` så funkade det (output från updateArticleCount)